packages/core-data/src/components/FacetTimeline.js

```diff
@@ -1,28 +1,502 @@
 // @flow
 
-import { Numbers, useTimer } from '@performant-software/shared-components';
-import * as Popover from '@radix-ui/react-popover';
-import * as Slider from '@radix-ui/react-slider';
+import { useTimer } from '@performant-software/shared-components';
 import { clsx } from 'clsx';
-import {
-  ChevronLeft,
-  ChevronRight,
-  RotateCcw,
-  ZoomIn,
-  ZoomOut
-} from 'lucide-react';
+import { scaleLinear, scaleTime } from 'd3-scale';
+import { timeTicks } from 'd3-time';
+import { RotateCcw, ZoomIn, ZoomOut } from 'lucide-react';
```

Comment:
Let's use or create icons we already have using the `Icon` component.

Reply:
The existing zoom in and out buttons are just a simple + and - and don't match [the designs](https://www.figma.com/design/5AWa1c3pJeVXwEeVPbRqZj/Place-Components-2.0?node-id=759-5326&t=lor5Kx96xRj9shNg-4), which have magnifying glasses. Should we create new icons for these in the `Icon` component, or reuse the simpler + and - ones for consistency across the design?

(on that note the `reset` icon we have is also in the opposite direction from the reset icon in the designs, but that seems less important…)

Reply:
From the development side, we're looking to move away from a dependency on third party icon libraries, since we now have our own `Icon` component. I'll defer to @cblissg on the design questions, but if we do want to use icons different from what we currently have, I think we should add new icons.